public/app/core/components/TransformersUI/configFromQuery/ConfigFromQueryTransformerEditor.test.tsx

```diff
@@ -38,7 +38,7 @@ describe('ConfigFromQueryTransformerEditor', () => {
   it('Should be able to select config frame by refId', async () => {
     setup();
 
-    let select = (await screen.findByText('Config query')).nextSibling!;
+    let select = (await screen.findByText('Config query')).nextSibling!.firstChild!;
```

Comment:
This change is needed because of the change in the InlineField component. the proper fix here would be to add `inputId` to Selelcts in `public/app/core/components/TransformersUI/configFromQuery/ConfigFromQueryTransformerEditor.tsx` to make them labeled (and accessible) but it required changes outside of the scope of this PR.